executor/write.go

```diff
@@ -158,8 +158,11 @@ func updateRecord(ctx sessionctx.Context, h int64, oldData, newData []types.Datu
 	} else {
 		sc.AddAffectedRows(1)
 	}
-
-	ctx.GetSessionVars().TxnCtx.UpdateDeltaForTable(t.Meta().ID, 0, 1)
+	colSize := make(map[int64]int64)
+	for id, col := range t.WritableCols() {
+		colSize[col.ID] = int64(len(newData[id].GetBytes()) - len(oldData[id].GetBytes()))
```

Comment:
`GetBytes` is only valid when the Kind of the Datum is KindString or KindBytes.

Maybe we need to add a method for Datum to return the size?